packages/cli/src/Version.ts

```diff
@@ -1,5 +1,5 @@
 import { enginesVersion, getCliQueryEngineBinaryType } from '@prisma/engines'
-import { getPlatform } from '@prisma/get-platform'
+import { getBinaryTargetForCurrentPlatform } from '@prisma/get-platform'
```

Comment:
package as well maybe?

Reply:
Let's keep this out of scope of this PR for now